PortalFetch/crawler_download.py

```diff
@@ -206,34 +207,42 @@ def waitUtilPageLoaded(driver, count):
     raise ElementNotVisibleException("Could not load the full page!")
 
 
-def generateQuarterAndFilename(quarterValue):
+def generateQuarterAndFilename(quarterValueStr):
     """Return quarter and filename.
 
     Args:
         quarterValue:the quarter_value in crawler.config
     Returns:
-        quarter str and filename str
+        quarter str list and filename str list
 
     """
-    year = quarterValue[0:4]
-    quarterSwitcher = {
-        "1": "Summer",
-        "2": "Fall",
-        "3": "Winter",
-        "4": "Spring",
-    }
-    schoolSwitcher = {
-        "1": "Foothill",
-        "2": "De Anza",
-    }
-    school = schoolSwitcher.get(quarterValue[5], "")
-    quarter = quarterSwitcher.get(quarterValue[4], "")
-    if quarter == "Summer":
-        year = str(int(year)-1)
-    quarterOutput = year + " " + quarter + " " + school
-    if school == "De Anza":
-        school = "De_Anza"
-    fileNameOutput = year + "_" + quarter + "_" + school + "_courseData.json"
+    n = 6
```

Comment:
avoid naming "n". the naming should be meaningful here. moreover, if a number is a constant, please use upper case. reference:https://www.python.org/dev/peps/pep-0008/#constants